typings-checker/queries.tsx

```diff
@@ -72,7 +72,7 @@ withFooAndBarProps = withFooAndBar.Props;
 declare const C2: React.ComponentType<typeof withFooAndBar.Props>;
 const WithFooAndBar = withFooAndBar(C2);
 
-// $ExpectError Property 'token' is missing in type '{}'.
+// $ExpectError Property 'token' is missing in type '{}' but required in type '{ token: string; }'.
```

Comment:
this should fix the CI but for some reason the old version is still run..

Reply:
also I'm not sure if we want to maintain this `type checks` in the future, do they really bring value?

Reply:
> these changes should fix the CI but for some reason, the old version of the file seems to still run..

meaning it doesn't fail on your machine but it fails on the CI without these changes? Looks like a different version of TS or related libraries issue?

> also I'm not sure if we want to maintain this type checks in the future, do they really bring value?

I would say yes until the `d.ts` are written manually - too easy to break them otherwise

Reply:
> meaning it doesn't fail on your machine but it fails on the CI without these changes? Looks like a different version of TS or related libraries issue?

yesterday the CI was failing with comments related to the "pre-changes" code, today everything works fine 👍